src/librbd/Journal.h

```diff
@@ -105,6 +105,10 @@ class Journal {
   static int get_tag_owner(ImageCtxT *image_ctx, std::string *mirror_uuid);
   static int get_tag_owner(librados::IoCtx& io_ctx, std::string& image_id,
                            std::string *mirror_uuid);
+  static void is_tag_owner(ImageCtxT *image_ctx, ContextWQ *work_queue,
+                           bool *is_tag_owner, Context *on_finish);
+  static void get_tag_owner(ImageCtxT *image_ctx, ContextWQ *work_queue,
```

Comment:
The async version of `get_tag_owner` is used by async `is_tag_owner` and this later is needed for async mirror enable/disable. Though as `get_tag_owner` is needed only for `is_tag_owner` I think I will hide it.